scripts/check-deps/src/getPackageConfig.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const {join, resolve} = require('path');
+
+function getPackageConfig(name) {
+	return require(resolve(join('packages', name, 'package.json')));
```

Comment:
Might want to rewrite this to use `fs.readFile` because it will trigger the [no-dynamic-require](https://github.com/liferay/eslint-config-liferay/blob/master/plugins/eslint-plugin-liferay/docs/rules/no-dynamic-require.md) rule once https://github.com/liferay/clay/pull/2536 gets merged (I left that one there for @matuzalemsteles to merge when he considered appropriate, but if it sits there too much longer it is going to get conflicts and have to be redone).

Reply:
Solved!